arlo-client/src/components/DataEntry/BallotAudit.tsx

```diff
@@ -127,34 +122,48 @@ const BallotAuditContest = ({
     })
   }
 
-  // Since the radio button values must be strings, not full objects,
+  // Since the checkbox button values must be strings, not full objects,
   // we condense our data model for interpretations to strings:
   // - For buttons representing interpretation VOTE (one for each
   // contest choice), the value is interpretation.choiceId
   // - For buttons representing other interpretations (BLANK,
   // CANT_AGREE), the value is interpretation.interpretation
-  const radioProps = {
-    name: `interpretation-${contest.name}`,
+  const checkboxProps = (value: string) => ({
+    name: `interpretation-${contest.name}-${value}`,
     handleChange: (e: React.ChangeEvent<HTMLInputElement>) => {
-      const { value } = e.currentTarget
+      const { checked } = e.currentTarget
       if (
         value === Interpretation.BLANK ||
         value === Interpretation.CANT_AGREE
       ) {
+        if (checked) {
+          setInterpretation({
+            ...interpretation,
+            interpretation: value,
+            choiceIds: [],
+          })
+        } else {
+          setInterpretation({
+            ...interpretation,
+            interpretation: null,
+            choiceIds: [],
+          })
+        }
```

Comment:
I got a little confused when I first read this because I couldn't tell what was different between these two cases lol. Maybe that's just me not reading closely.

I think it could be a bit more obvious what's different if you wrote it like this, but not a big deal

```ts
setInterpretation({
  ...interpretation,
  interpretation: checked ? value : null,
  choiceIds: [],
})
```

Reply:
@jonahkagan Yeah, that's a much better way to do it, haha.